.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+.src/resources/credentials.env
```

Comment:
The line `.src/resources/credentials.env` seems to have a typo. The leading dot before `src` is likely incorrect unless the directory is actually named `.src`. If the intention is to ignore `src/resources/credentials.env`, it should be written as `src/resources/credentials.env`.